README.md

```diff
@@ -101,7 +101,6 @@ The plugin will have made the following tasks available to your project:
 | libertyJavaDump | Dumps diagnostic information from the Liberty Profile server JVM. | 
 | libertyDebug | Runs the Liberty Profile server in the console foreground after a debugger connects to the debug port (default: 7777). | 
 | libertyStatus | Checks the WebSphere Liberty Profile server is running. |
-| libertyPackage | Generates a WebSphere Liberty Profile server archive. |
```

Comment:
Consider splitting out into separate documents like Liberty Maven plug-in and give gradle.build sniplet example.